virtool/auth/utils.py

```diff
@@ -0,0 +1,93 @@
+from logging import getLogger
+
+import openfga_sdk
+from aiohttp import ClientConnectorError
+from openfga_sdk import (
+    CreateStoreRequest,
+    WriteAuthorizationModelRequest,
+    TypeDefinition,
+    Userset,
+)
+from openfga_sdk.api import open_fga_api
+
+logger = getLogger("OpenFGA")
+
+
+async def connect_openfga():
+    configuration = openfga_sdk.Configuration(
+        api_scheme="http", api_host="localhost:8080"
```

Comment:
OpenFGA is not always going to be running on `localhost`. This will even be a problem in Docker Compose where each service has its own IP and DNS name within the Docker network.

These values need to be configurable via CLI and have reasonable defaults.